test/taskcreator_test.js

```diff
@@ -207,6 +253,72 @@ suite('taskcreator_test.js', function() {
           (err) => { debug('Got expected error: %s', err); });
       });
     }
+
+    test('adds a new row to lastFire', async function() {
+      let hook = _.cloneDeep(defaultHook);
+      let taskCreateTime = new Date();
+      await creator.appendLastFire({
+        hookId: hook.hookId,
+        hookGroupId: hook.hookGroupId,
+        firedBy: 'test', 
+        result: 'success', 
+        taskId: hook.nextTaskId,
+        taskCreateTime,
+        result: 'success',
+        error: '',
+      }
+      );
+
+      const res = await helper.LastFire.load({
+        hookGroupId: hook.hookGroupId,
+        hookId: hook.hookId,
+        taskId:  hook.nextTaskId,
+      });
+      assume(res.taskId).equals(hook.nextTaskId);
+    });
+
+    test('Fetch two appended lastFire rows independently', async function() {
+      let hook = _.cloneDeep(defaultHook);
+      let hook2 = _.cloneDeep(defaultHook2);
```

Comment:
In fact, maybe we don't even need a variable named `defaultHook2` -- the `_.assign(..)` I mentioned above could appear directly here.